docs/source/releasenotes.rst

```diff
@@ -1,9 +1,66 @@
 Release notes
 =============
 
+.. note::
+  The support for Python version 3.6 has been **REMOVED** from the ``rpaframework``
+  packages starting with the following versions:
+    - ``rpaframework`` **14.0.0**
+    - ``rpaframework-aws`` **2.0.0**
```

Comment:
Don't we want to include `rpaframework-core` **8.0.0** as well here? (if we still want to do a major release for this one too @mikahanninen )